charts/feature-cluster-metrics/templates/_helpers_modules.tpl

```diff
@@ -0,0 +1,24 @@
+{{/* Loads a module from Alloy Modules*/}}
+{{/* Inputs: . ($), name (string), path (string)*/}}
+{{- define "alloyModules.load" }}
+{{- if eq .Values.global.alloyModules.source "configMap" }}
+{{- $pathParts := regexSplit "/" .path -1 }}
+{{- $configMapName := printf "%s-alloy-module-%s" $.Release.Name (index $pathParts 1) }}
+{{- $moduleFile := (slice $pathParts 2) | join "_" }}
+remote.kubernetes.configmap {{ .name | quote }} {
+  name = {{ $configMapName | quote }}
+  namespace = {{ .Release.Namespace | quote }}
+}
+
+import.string {{ .name | quote }} {
+  content = remote.kubernetes.configmap.{{ .name }}.data[{{ $moduleFile | quote }}]
```

Comment:
`$moduleFile` is presumably a path visible to the Alloy pod used to load the module from disk?

Reply:
Yep. `remote.kubernetes.configmap` creates an array of string objects, referenced by their keys.